src/internal/views/nunjucks/gauging-stations/licences.njk

```diff
@@ -0,0 +1,221 @@
+
+                <td class="govuk-table__cell ">
+                  {% for tag in licenceTags %}
+                    {{ tag.abstractionPeriod | abstractionPeriod }}<br>
+                  {% endfor %}
+                </td>
+                <td class="govuk-table__cell ">
+                  <!--loop through conditions for that licence-->
+                  {% for tag in licenceTags %}
+                    {{tag.conditionType |capitalize}}<br>
+                    {{tag.comstatus}}<br>          
+                  {% endfor %}
+                </td>
+                <td class="govuk-table__cell govuk-table__cell--numeric">
+                  <!--loop through the tag values and if they are a flow value add them to this cell-->
+                  {% for tag in licenceTags %}
+                    {{tag.thresholdValue}}<br>
+                  {% endfor %}
+                </td>
+                <td class="govuk-table__cell ">
+                  <!--loop through the tag values and if they are a flow value add them to this cell  | units |safe -->
+                  {% for tag in licenceTags %}
+                    {{tag.thresholdUnit}}<br>
+                  {% endfor %}
+                </td>
+                {% for licence in station['licences']%}
+                  {% if licence.number == i.licenceNumber %}
+                    {% if station.licences[loop.index0]['communications'].length %}
+                      {% set communications = station.licences[loop.index0]['communications'] %}
+                      <!-- type.split(" ")[0] | capitalize -->
+                      <td class="govuk-table__cell ">
+                        {% set commsType = '' + communications[0].type.trim().replace(' ','_') %}
+                        {{ badge({status:[commsType]} | gaugingStationBadge(false) ) }}
+                      </td>
+                      <td class="govuk-table__cell">  {{communications[0].sent}}</td>
+                    {% else %}
+                      {% set communications = "" %}
+                      <td class="govuk-table__cell"> </td>
+                      <td class="govuk-table__cell"> </td>
+                    {% endif %}
+
+                  {% endif %}
+                {% endfor %}
+              </tr>
+              {% endfor %}
+
+            </tbody>
+          </table>
+
+          <div class="govuk-button-group">
+            {{ govukButton({
+              text: "Tag a licence",
+              href: "monitoring-stations/" + gaugingStationId + "/new-tag",
+              classes: "govuk-button--secondary"
+            }) }}
+
+            {{ govukButton({
+              text: "Remove a tag",
+              href: "monitoring-stations/" + gaugingStationId + "/remove-tag",
+              classes: "govuk-button--secondary"
+            }) }}
+          </div>
+
+        {% else %}
+          <p class="govuk-body">There are no licences tagged with restrictions for this monitoring station</p>
+          {{ govukButton({
```

Comment:
Additionally, I think you're missing a forward slash at the beginning of the href. Should probably be /monitoring-stations/ rather than monitoring-stations/

Could I please also ask you to change `new-tag` to `tagging-licence`